app/models/exercise.rb

```diff
@@ -5,20 +5,24 @@ class Exercise < ApplicationRecord
           WithAssignments,
           FriendlyName,
           WithLanguage,
-          Assistable
+          Assistable,
+          WithRandomizations
 
   include Submittable,
           Questionable
 
   include SiblingsNavigation,
           ParentNavigation
 
+  serialize :randomizations, Hash
```

Comment:
This `serialize` could go into `WithRandomizations`

Reply:
I left it out in case one wanted to name the randomizations method something different. That's why I'm also passing it as a parameter to interpolations_of, would you like me to hard code that?

Reply:
Yeah, I would not generalize to scenarios that don't yet exist.